src/test/java/Homework16.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public static void Navigation() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void navigation() `

Reply:
When I make this change, though, it no longer recognizes Homework16 as a test. Can you explain why?